python/tests/stress/arcticdb/version_store/test_mem_leaks.py

```diff
@@ -185,6 +187,7 @@ def proc_to_examine():
 
     check_process_memory_leaks(proc_to_examine, 25, max_mem_bytes, 80.0)
 
+@SKIP_CONDA_MARK
 @pytest.mark.skipif(sys.platform == "darwin", reason="Problem on MacOs")
```

Comment:
the skipif for `darwin `is a bit redundant because of the conda mark, consider removing it.
Also I see that the other test (`test_mem_leak_read_all_arctic_lib`) is skipped on windows, maybe we should skip this one as well proactively, especially as all of the other tests that use the `lmdb_version_store_very_big_map` fixture are skipped at the moment.